common/lib/xmodule/xmodule/templates/problem/circuitschematic.yaml

```diff
@@ -62,15 +61,23 @@ data: |
         correct = ['incorrect']
       </answer>
       </schematicresponse>
-      </fieldset>
     
-          <solution>
-              <div class="detailed-solution">
-                  <p>Explanation</p>
-                  <p>You can form a voltage divider that evenly divides the input voltage with two identically valued resistors, with the sampled voltage taken in between the two.</p>
-                  <p><img src="/static/images/voltage_divider.png"/></p>
-                  <p>You can form a simple high-pass filter without any further constraints by simply putting a resistor in series with a capacitor. The actual values of the components do not really matter in this problem.</p>
-                  <p><img src="/static/images/high_pass_filter.png"/></p>
-              </div>
-          </solution>
+      <solution>
+          <div class="detailed-solution">
+              <p>Explanation</p>
+              <p>
+              You can form a voltage divider that evenly divides the input
+              voltage with two identically valued resistors, with the sampled
+              voltage taken in between the two. 
+              </p>
+              <p><img src="/static/images/voltage_divider.png"/></p>
```

Comment:
@mhoeber because the text that precedes it does a pretty good job of describing what this graphic shows, an empty alt attribute is acceptable, but we at least need an empty ```alt=""```  Same applies to the high_pass_filter below.